src/config.h.in

```diff
@@ -0,0 +1,3 @@
+#if XAPIAN_MAJOR_VERSION >= 2 || (XAPIAN_MINOR_VERSION >= 3 && XAPIAN_REVISION >= 6)
+#define XAPIAN_RANGE_PROCESSOR_SUPPORT
+#endif
```

Comment:
It's a good idea to have a trailing newline in every file, or future diffs at the end don't work properly.